core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/workflow/WorkflowCompiler.scala

```diff
@@ -95,21 +91,21 @@ class WorkflowCompiler(val workflowInfo: WorkflowInfo, val context: WorkflowCont
           case BreakpointCondition.NOT_CONTAINS =>
             tuple => !tuple.getField(column).toString.trim.contains(conditionBp.value)
         }
-        controller ! PassBreakpointTo(
-          operatorID,
-          new ConditionalGlobalBreakpoint(
-            breakpointID,
-            tuple => {
-              val texeraTuple = tuple.asInstanceOf[Tuple]
-              predicate.apply(texeraTuple)
-            }
-          )
-        )
+//        controller ! PassBreakpointTo(
+//          operatorID,
+//          new ConditionalGlobalBreakpoint(
```

Comment:
Do we need this code in commented state?

Reply:
We don't, but I will do the clean up on the next PR.